numpy/ctypeslib.pyi

```diff
@@ -0,0 +1,8 @@
+from typing import Any
+
+load_library: Any
+ndpointer: Any
+ctypes_load_library: Any
```

Comment:
Given this has been deprecated for at least 14 years, I think we can skip it here.

https://github.com/numpy/numpy/blame/64619c55896dad1927afe78c0702a58abf0d038f/numpy/ctypeslib.py#L157

Reply:
Removed entirely in #17116